Expensify/App
# Added persistent Created Room Header to the Chat

### Details

We needed to replace the initial Room header with a persistent room header. I have added a new component that renders when action-type is `CREATED`. 
I have also updated the names of a component and styles  to reflect the change (Hope that's fine).
### Fixed Issues

$ https://github.com/Expensify/App/issues/7077

### Tests

1. Open a new/existing chat.
2. Verify the position of the header and scroll-ability within the chat.

### QA Steps

1. Open a new/existing chat.
2. Verify the position of the header and scroll-ability within the chat.
- [X ] Verify that no errors appear in the JS console

### Tested On


### Screenshots


#### Web


https://user-images.githubusercontent.com/48352577/151380917-45e1dabb-a7d8-4aff-8338-3c2c5d8880ec.mp4


#### Mobile Web

![WhatsApp Image 2022-01-28 at 4 36 22 PM](https://user-images.githubusercontent.com/48352577/151537683-6e679794-0ce9-453f-8ac8-6daf2b3e5794.jpeg)



#### Desktop


#### iOS


#### Android

![WhatsApp Image 2022-01-28 at 4 36 23 PM](https://user-images.githubusercontent.com/48352577/151537701-750cb450-36b0-4c89-8f5b-92c3f4b86bdf.jpeg)
